frontends/election-manager/src/components/format_usb_modal.tsx

```diff
@@ -0,0 +1,178 @@
+    return (
+      <Modal
+        content={
+          <Prose>
+            <h1>Failed to Format USB Drive</h1>
+            <p>Failed to format USB drive: {errorMessage}</p>
+          </Prose>
+        }
+        onOverlayClick={onClose}
+        actions={<Button onPress={onClose}>Close</Button>}
+      />
+    );
+  }
+
+  if (state === 'done') {
+    return (
+      <Modal
+        content={
+          <Prose>
+            <h1>USB Drive Formatted</h1>
+            <p>
+              USB drive successfully re-formatted. It is now ready to use with
+              VotingWorks devices.
+            </p>
+          </Prose>
+        }
+        onOverlayClick={onClose}
+        actions={<LinkButton onPress={onClose}>Close</LinkButton>}
+      />
+    );
+  }
+
+  if (state === 'formatting') {
+    return <Modal content={<Loading>Formatting USB Drive</Loading>} />;
+  }
+
+  if (state === 'confirm') {
+    return (
+      <Modal
+        content={
+          <Prose>
+            <h1>Confirm Format USB Drive</h1>
+            <p>
+              <strong>Warning:</strong> formatting will delete all files on the
+              USB drive. Back up USB drive files before formatting.
+            </p>
+          </Prose>
+        }
+        onOverlayClick={onClose}
+        actions={
+          <React.Fragment>
+            <Button primary onPress={format}>
+              Format USB
+            </Button>
+            <Button onPress={onClose}>Close</Button>
+          </React.Fragment>
+        }
+      />
+    );
+  }
+
+  switch (usbDrive.status) {
+    case 'absent':
+      return (
+        <Modal
+          content={
+            <Prose>
+              <h1>No USB Drive Detected</h1>
+              <p>
+                <UsbImage />
+                Insert a USB drive you would like to format.
+              </p>
+            </Prose>
+          }
+          onOverlayClick={onClose}
+          actions={<Button onPress={onClose}>Cancel</Button>}
+        />
+      );
+    case 'ejecting':
+    case 'mounting':
+      return (
+        <Modal
+          content={<Loading />}
+          onOverlayClick={onClose}
+          actions={<LinkButton onPress={onClose}>Cancel</LinkButton>}
+        />
+      );
+    case 'ejected':
+    case 'bad_format':
+    case 'mounted': {
+      return (
+        <Modal
+          content={
+            <Prose>
+              <h1>Format USB Drive</h1>
+              <p>
+                {usbDrive.status === 'bad_format'
+                  ? 'The format of the inserted USB drive is not VotingWorks compatible. Would you like to format the USB drive?'
+                  : 'The format of the inserted USB drive is already VotingWorks compatible. Would you like to re-format the USB drive?'}
+              </p>
+            </Prose>
+          }
+          onOverlayClick={onClose}
+          actions={
+            <React.Fragment>
+              <Button primary onPress={() => setState('confirm')}>
+                Format USB
```

Comment:
Maybe "Format USB Drive" to be consistent with the rest of the modal

Reply:
I'll leave the buttons as-is, I prefer to keep the buttons less wordy but that is just a personal preference.